libs/partners/google-genai/langchain_google_genai/chat_models.py

```diff
@@ -314,6 +314,8 @@ def _parse_chat_history(
             role = "model"
         elif isinstance(message, HumanMessage):
             role = "user"
+        elif isinstance(message, ChatMessage):
+            role = message.role
```

Comment:
what value can `message.role` have? should we maybe map them to "model" / "user" somehow? 
or at least check and throw a meaningful error?